exchange_calendars/exchange_calendar_xtai.py

```diff
@@ -111,7 +132,7 @@ def weekend_makeup(dt: datetime.datetime) -> datetime.datetime:
     return dt
 
 
-def bridge_mon(dt: datetime.datetime) -> datetime.datetime | None:
+def bridge_mon(dt: datetime.datetime, checker: Callable = check_after_2013) -> datetime.datetime | None:
```

Comment:
Could you revise the documentation to reflect the new implementation, adding a 'Parameters' section and including the new 'checker' parameter to it.

Also, the linting issue is in line 147. Removing the empty line should resolve it.